.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.formatting.provider": "black"
+}
```

Comment:
issue (blocking): if we're going to use autoformatting, would be helpful if it weren't dependent upon a specific editor config.

can we integrate this formatter into `bin/fmt`?

Reply:
yeah let me take this out i unenabled it on my vscode cuz it was not quite right